includes/discovery/sensors/voltage/dell.inc.php

```diff
@@ -35,7 +35,7 @@
         $warnlimit = $entry['voltageProbeUpperNonCriticalThreshold'] / $divisor;
         $limit = $entry['voltageProbeUpperCriticalThreshold'] / $divisor;
 
-        discover_sensor($valid['sensor'], 'voltage', $device, $cur_oid . $index, $index, 'dell', $descr, $divisor, '1', $lowlimit, $low_warn_limit, $warnlimit, $limit, $value, 'snmp', $index);
```

Comment:
looking at the code, this one `$index` is probably useless (because this table voltageProbeTable is not index with ENTITY-MIB entPhysicalIndex at all), and your fix correct. But as we don't have test data to confirm, I would suggest to revert it.